heat/classification/tests/test_knn.py

```diff
@@ -84,24 +74,14 @@ def test_fit_one_hot(
     ):
         x = ht.load_hdf5("heat/datasets/iris.h5", dataset="data")
 
-        # keys as label array
-        keys = []
-        for i in range(50):
-            keys.append(0)
-        for i in range(50, 100):
-            keys.append(1)
-        for i in range(100, 150):
-            keys.append(2)
+        keys = [0 for _ in range(50)]
+        keys.extend(1 for _ in range(50, 100))
+        keys.extend(2 for _ in range(100, 150))
```

Comment:
see above

Reply:
here you must initialize the labels with `split=0`

Reply:
@ClaudiaComito mam, here also do you mean to say this `keys = ht.zeros(150, split=0, dtype=ht.int32)`? because labels is already initialized with `split=0` like this `labels = ht.array(keys, split=0)`